src/itest/kotlin/nl/lifely/zac/itest/config/ItestConfiguration.kt

```diff
@@ -17,15 +18,16 @@ object ItestConfiguration {
     const val KEYCLOAK_HOSTNAME_URL = "http://localhost:8081"
     const val KEYCLOAK_HEALTH_READY_URL = "$KEYCLOAK_HOSTNAME_URL/health/ready"
     const val KEYCLOAK_REALM = "zaakafhandelcomponent"
-    const val KEYCLOAK_CLIENT = "zaakafhandelcomponent"
-    const val KEYCLOAK_CLIENT_SECRET = "keycloakZaakafhandelcomponentClientSecret"
+    const val ZAC_KEYCLOAK_CLIENT = "zaakafhandelcomponent"
```

Comment:
I would name these `KEYCLOAK_ZAC_CLIENT_ID` and `KEYCLOAK_ZAC_CLIENT_SECRET`, as this is about the client configured in KEYCLOAK for ZAC.